podkrepi-bg/frontend
# Added button to donation success page

Closes #1174

Users do not have a clear way to go back to the campaign after a successful donation.

## Motivation and context

- added new button "back to campaign" on the donation success page
- added link to the campaign title

## Screenshots:

Before|After
---|---
![image](https://user-images.githubusercontent.com/893608/203746935-a4f50761-0c48-4269-86af-f01f41176bb3.png)|![image](https://user-images.githubusercontent.com/893608/203746994-e5e63fb2-723b-4290-8745-3c42786b4b70.png)


## Testing

1. Create a successful campaign
2. See the title of the campaign as a link to the campaign
3. See a new button "See the campaign" as a link to campaign
